src/assets/HTMLAsset.js

```diff
@@ -46,6 +48,9 @@ class HTMLAsset extends Asset {
         for (let attr in node.attrs) {
           let elements = ATTRS[attr];
           if (elements && elements.includes(node.tag)) {
+            if (node.tag === 'meta' && !isFile(node.attrs[attr])) {
```

Comment:
what does the `isFile` check fix that `isUrl` doesn't already check for?

Reply:
It can also be a local file with isFile, meta tags can contain a lot more than just locations so i added a check to make sure it doesn't try to include other meta content

Reply:
Seems like this would break for anything that has a `.` in it: e.g. `<meta name="something" content="a.b.c">`. I can't think of a good workaround. We may need to maintain a list of known meta tag names...